subsys/bluetooth/controller/hci_driver.c

```diff
@@ -682,8 +681,8 @@ static int configure_memory_usage(void)
 	if (required_memory < 0) {
 		return required_memory;
 	}
-#endif
 
+#if defined(CONFIG_BT_PERIPHERAL) || defined(CONFIG_BT_CENTRAL)
```

Comment:
I don't really understand why we need this. SDC_CFG_TYPE_BUFFER_CFG and SDC_CFG_TYPE_EVENT_LENGTH are available on all variants/archives -- also, their values have no effect on the RAM usage in sdc.c if the central and peripheral counts are 0 (see s_pdu_init())

Reply:
Yeah, I double checked.. now witht he other hunk where we set peripheral_count.count to 0, we dont need this hunk to get the original test passing. So I removed this hunk.